ui/pages/SignIn.tsx

```diff
@@ -62,12 +62,16 @@ const DocsWrapper = styled(Flex)`
 function SignIn() {
   const flags = useFeatureFlags();
 
+  const { loading, error } = React.useContext(FeatureFlags);
+
   if (flags.WEAVE_GITOPS_AUTH_ENABLED === false) {
     return <Redirect to="applications" />;
   }
 
   const formRef = React.useRef<HTMLFormElement>();
-  const { signIn, error, loading } = React.useContext(Auth);
+  const { signIn } = React.useContext(Auth);
+  const authLoading = React.useContext(Auth).loading;
+  const authError = React.useContext(Auth).error;
```

Comment:
Something like this could work?

```suggestion
  const { signIn, error: authError, loading: authLoading } = React.useContext(Auth);
```